include/cuco/detail/open_addressing/open_addressing_ref_impl.cuh

```diff
@@ -278,17 +278,17 @@ class open_addressing_ref_impl {
    *
    * @return The bucket extent.
    */
-  [[nodiscard]] __host__ __device__ constexpr extent_type bucket_extent() const noexcept
+  [[nodiscard]] __host__ __device__ constexpr extent_type extent() const noexcept
   {
-    return storage_ref_.bucket_extent();
+    return storage_ref_.extent();
   }
 
   /**
    * @brief Returns a const_iterator to one past the last slot.
    *
    * @return A const_iterator to one past the last slot
    */
-  [[nodiscard]] __host__ __device__ constexpr const_iterator end() const noexcept
+  [[nodiscard]] __host__ __device__ constexpr iterator end() const noexcept
```

Comment:
Why are we switching from `const_iterator` to `iterator`? Also, the docstring should then say `iterator` instead of `const_iterator`.